tests/unit/test_imbalance.py

```diff
@@ -19,11 +19,11 @@ def test_result_tfs(_tfs_file):
     res = merge_kmod_results.get_lumi_imbalance(_tfs_file)
 
     assert res["imbalance"] == 0.974139299943968
-    assert res["relative_error"] == 0.003859317636164786
-    assert res["eff_beta_ip1"] == 0.39800399369855577
-    assert res["rel_error_ip1"] == 0.0015434779687894644
-    assert res["eff_beta_ip5"] == 0.4085698972636139
-    assert res["rel_error_ip5"] == 0.0023158396673753213
+    assert res["relative_error"] == 0.0027110906825478205
+    assert res["eff_beta_ip1"] == 0.3980039936985557
+    assert res["rel_error_ip1"] == 0.0006143103957639417
+    assert res["eff_beta_ip5"] == 0.40856989726361387
+    assert res["rel_error_ip5"] == 0.0009461823749785368
 
 
```

Comment:
Should be renamed to `test_merge_kmod_results.py`  btw

Reply:
I couldn't put comments on the lines, but: 
```
RESULTS = "results.tfs"
LSA_RESULTS = "lsa_results.tfs"
```
should be imported from the script.

-----------

we should have one test that calls the function from commandline to see if the entrypoint-parameters were set properly

Reply:
there is already one tests with a call from cmdline ..